package/clip.py

```diff
@@ -5,14 +5,11 @@
 from datetime import datetime
 import glob
 def display(snippet_name, password):
-    # Retrieve the current time in HHMM format
     current_time = datetime.now().strftime("%H%M")
     
-    # Check if the provided password matches the current time
-    if str(password) != current_time:
+    if int(password) != current_time:
```

Comment:
This condition will never be true. as you're comparing int `password` with a string `current_time`
Consider typecasting `current_time` to int, OR don't typecast `password` 

```suggestion
    if int(password) != int(current_time):
```

OR Consider this:
```suggestion
    if password != current_time:
```